inst/stan/epinowcast.stan

```diff
@@ -76,6 +79,9 @@ parameters {
   vector<lower=0>[neff_sds] logmean_sd; // pooled modifiers to logmean
   vector<lower=0>[neff_sds] logsd_sd; // pooled modifiers to logsd
   vector<lower=0>[nrd_eff_sds] rd_eff_sd; // pooled modifiers to report date
+  real alpha_start[g]; // starting value for alpha
+  real<lower=0> alpha_sd; // standard deviation of the random walk increments
```

Comment:
This implementation assumes the same scale of the random walk noise for all groups. Should we change this to be individual for each group? If yes, fixed or random effect? I know this may be taken over by a formula interface in the long term...

Reply:
Let's leave as the simple case for now as hopefully we will be ready to update to the formula interface in the short to medium term.